apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -90,6 +116,32 @@ public boolean isEnabled() {
                     return Crashes.isEnabled().get();
                 }
             });
+            initChangeableSetting(R.string.appcenter_crashes_text_attachment_key, getCrashesTextAttachmentSummary(), new Preference.OnPreferenceChangeListener() {
+                @Override
+                public boolean onPreferenceChange(Preference preference, Object newValue) {
+                    setKeyValue(TEXT_ATTACHMENT_KEY, (String) newValue);
+                    sCrashesListener.setTextAttachment((String) newValue);
+                    preference.setSummary(getCrashesTextAttachmentSummary());
+                    return true;
+                }
+            });
+            initClickableSetting(R.string.appcenter_crashes_file_attachment_key, getCrashesFileAttachmentSummary(), new Preference.OnPreferenceClickListener() {
+                @Override
+                public boolean onPreferenceClick(Preference preference) {
+                    Intent intent;
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+                        intent = new Intent(Intent.ACTION_OPEN_DOCUMENT);
+                        intent.addCategory(Intent.CATEGORY_OPENABLE);
```

Comment:
would that be easy to focus the downloads directory by default?

Reply:
On galaxy nexus (android 4.0.3) it offers only photos, contacts or music also.

Reply:
really optional comments, you can ignore that if it's complex.

Reply:
Actually, I don't think it will improve the user experience, so I prefer to leave it as is. I guess that's not big deal anyway.